packages/react-router-redux/package-lock.json

```diff
@@ -12,21 +12,21 @@
 			"resolved": "https://registry.npmjs.org/encoding/-/encoding-0.1.12.tgz",
 			"integrity": "sha1-U4tm8+5izRq1HsMjgp0flIDHS+s=",
 			"requires": {
-				"iconv-lite": "~0.4.13"
+				"iconv-lite": "0.4.19"
```

Comment:
Based on these lines, it appears you're using an older version of npm. Update to npm@6.4.0 and re-run these installs. That should cut down on the number of changes here.

Reply:
Ah, I was wondering what was going on here. I tried both 8 and 9, but didn't go to 10. 👍